docker_images/mbed-os-env/README.md

```diff
@@ -0,0 +1,49 @@
+# mbed-OS development environment docker image
+
+This docker image is the official mbed-OS development environment.
+* It is based on ubuntu 20.04
+* Python and cmake are installed
+* arm-none-eabi-gcc toolchain is installed
+* Latest released version of mbed-cli and GreenTea are installed
+* All other mbed-OS dependency tools are installed.
+
+# How to use docker image:
+
+## Pull docker images
+```bash
+docker pull mbedos/mbed-os-env
+
+```
+
+## Run mbed OS environment without HW support (build mbed images only)
+launch docker by
+```bash
+docker run -it mbedos/mbed-os-env:<lable>
+```
+Then you will have the container with mbed OS development environment.
+you should be able to compile mbed commands/examples as recommenced in mbed documentations
+e.g.
+```bash
+mbed import mbed-os-example-blinky
+cd mbed-os-example-blinky
+mbed compile -m <TARGET> -t GCC_ARM
+```
+
+## Run mbed OS environment with HW support (USB pass-through)
+If you want to use this docker container connect and flash your targets. you need some extra command line option to pass-through your USB devices.
+```bash
+sudo docker run -it --privileged -v /dev/disk/by-id:/dev/disk/by-id -v /dev/serial/by-id:/dev/serial/by-id mbed-os-env:<label>
```

Comment:
`privileged` is not mandatory. I will search for provide details. It is important to point out that it only works with a linux host. It doesn't work on Windows or Mac OS